systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -344,4 +347,75 @@ public void testKRaftMode(ExtensionContext extensionContext) {
         LOGGER.info("Waiting for clients to finish sending/receiving messages.");
         ClientUtils.waitForClientsSuccess(producerName, consumerName, INFRA_NAMESPACE, MESSAGE_COUNT);
     }
+    @IsolatedTest
+    void testSwitchingConnectStabilityIdentifiesFeatureGateOnAndOff(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final TestStorage testStorage = new TestStorage(extensionContext);
+        final int connectReplicas = 1;
+        final int messageCount = 500;
+        List<EnvVar> coEnvVars = new ArrayList<>();
+
+        coEnvVars.add(new EnvVar(Environment.STRIMZI_FEATURE_GATES_ENV, "", null));
```

Comment:
If you make it more explicit and disable it here, enable later, disable later, the test will be more future proof as the Feature Gate progresses through the different stages.